query07.sql

```diff
@@ -2,5 +2,15 @@
   How many trips started on one day and ended in the next?
 */
 
--- Enter your SQL query here
-select ...
+-- For 2019Q2
+SELECT COUNT(extract(day from end_time::timestamp - start_time::timestamp))
```

Comment:
This gets the correct number, but note that you don't have to `count` the same expression you're filtering by. You could just use `count(*)` instead of `count(extract(day from end_time::timestamp - start_time::timestamp))`.